release.json

```diff
@@ -18,8 +18,8 @@
     },
     "nightly-a7": {
         "INTEGRATIONS_CORE_VERSION": "master",
-        "OMNIBUS_SOFTWARE_VERSION": "master",
-        "OMNIBUS_RUBY_VERSION": "datadog-5.5.0",
+        "OMNIBUS_SOFTWARE_VERSION": "chouquette/config_guess",
+        "OMNIBUS_RUBY_VERSION": "chouquette/config_guess",
```

Comment:
Same as above

Reply:
Ah, that's why I initially opened it as a draft, and I forgot about it after Pierre mentionned it :facepalm:

Reply:
Thanks, without your nit I'd probably have commited the change as is :sweat_smile: